armi/nuclearDataIO/tests/test_xsLibraries.py

```diff
@@ -311,6 +311,7 @@ def tearDownClass(cls):
         del cls.libLumped
 
     def setUp(self):
+        super().setUp()
```

Comment:
I don't like super but it makes sense if it works.

Reply:
https://github.com/terrapower/armi/pull/1567/commits/6aff1d6e846bde953d7e66960f26f3f30ef37082 🙈